static/app/components/dropdownMenu.tsx

```diff
@@ -118,15 +118,20 @@ function DropdownMenu({
     [menuProps, hasFocus]
   );
 
+  const showDividers = useMemo(
```

Comment:
No need for a useMemo here since it returns a primitive and the operation is quick